### ISSUE TITLE
DHCP.pm 17.2: strange entry name added for hostname 'x-y.dom.ain' 

### ISSUE THREAD

**Issue by jouvin** [quattor/aii#273]
I just realized that with 17.2 (but may be the case since a long time), if you have a host name like `x-y.dom.ain`, the `host` entry added in the DHCP configuration will be `x.dom.ain-y`. This is basically harmless but a litle bit surprising... Is it a design choice or a bug?

If this is a bug, when fixing it, it should continue to search for the buggy name when deleting a name to allow an easy cleanup of the problem...

**Comment by jrha** [comment]
Are you sure, we don't see this...
```
 host ceph-dev-sn1.gridpp.rl.ac.uk {  # added by aii-dhcp
```

```sh
# rpm -qa | grep aii
aii-custom-1.0.0-1.noarch
aii-server-17.2.0-1.noarch
ncm-aiiserver-17.2.0-1.noarch
aii-pxelinux-17.2.0-1.noarch
aii-ks-17.2.0-1.noarch
aii-dhcp-17.2.0-1.noarch
```

**Comment by jouvin** [comment]
Yes I'm sure... I've to check if there is something in our config that could result in this...
